### PR TITLE
Added Pascal triangle using Recursion

### PR DESCRIPTION
**Pull request by SwarupKMondal**
## Related Issue
- Given an integer x print the first x rows of pascal's triangle(#1180 )

## Proposed Changes
-  Explanation
-  Code
-  Time Complexity & space Complexity

## Additional Info
- Link to the question : https://www.geeksforgeeks.org/pascal-triangle/

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1  
Screenshot (312)

### CODE REVIEW

**Comment by Mrjoy832** [comment]
Resolve branch conflicts.....

**Comment by Mrjoy832** [comment]
Put tick on the required boxes only

**Comment by Mrjoy832** [comment]
Why 5 files changed? delete unnecessary files

**Comment by SwarupKMondal** [comment]
done Sir

**Comment by Mrjoy832** [comment]
Resolve conflict

**Comment by SwarupKMondal** [comment]
done sir

File: R-Recursion/README.md [hunk 1]
```diff
@@ -449,6 +449,26 @@ Output:
...
+- Time Complexity :- O(n)
+- Space Complexity :- O(1)
+
+
+## Advantage :
+The main advantage lies in the fact that you can pre-compute all (ni ki) where ni < n and ki <= k in O(n^2) time.
+
+
+
+=======
```
Comment (Mrjoy832) [inline R-Recursion/README.md:471]: Remove the hand written image, it's not clear add fresh one
**readme should contain**
- problem statement
- example
- image
- algo
- time&space
- advantage&dis
**add index at top**

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**